morgoth/addons/api/views.py

```diff
@@ -31,6 +31,21 @@ class AddonGroupViewSet(ModelViewSet):
         permissions.DjangoModelPermissionsOrAnonReadOnly,
     ]
 
+    @list_route(methods=['POST'])
+    def update_built_in(self, request, *args, **kwargs):
+        browser_version = request.data.get('browser_version')
+        group, _ = AddonGroup.objects.get_or_create(browser_version=browser_version)
+
+        group.built_in_addons.clear()
+
+        if 'addons' in request.data:
```

Comment:
I think this means that not including the `addons` parameter will clear out the list of addons, but I'd rather have `addons` be required, and only clear out the list if it's explicitly empty.